examples/helloworld/tf_example1/README.md

```diff
@@ -30,12 +30,14 @@ python test.py --dataset_location=/path/to/imagenet/
 ### 2. Introduction 
 We only need to add the following lines for quantization to create an int8 model.
 ```python
+    top1 = TensorflowTopK(k=1)
     config = PostTrainingQuantConfig(calibration_sampling_size=[20])
     q_model = fit(
         model="./mobilenet_v1_1.0_224_frozen.pb",
         conf=config,
         calib_dataloader=calib_dataloader,
-        eval_dataloader=eval_dataloader)
+        eval_dataloader=eval_dataloader,
+        eval_metric=top1)
```

Comment:
why this issue just gets raised now?

Reply:
in pythonic_config, there is default value for eval_metric.